### DISCUSSION TITLE
api调用

### DISCUSSION THREAD

**Discussion by YSblack** [open-webui/open-webui#2043] [General]
无法进行api调用

![image](https://github.com/open-webui/open-webui/assets/44052382/8f7b1a50-2411-445b-a342-63f0a62fca6a)
![image](https://github.com/open-webui/open-webui/assets/44052382/804d5aa2-83bd-4912-97fa-f94dc05b60b3)
![image](https://github.com/open-webui/open-webui/assets/44052382/fc7e2f38-c66e-43fb-a18f-b6b013009c43)

**Comment by yutaomou** [comment]
同样遇到这个问题
